src/utilities/EmpireAnalysis.ts

```diff
@@ -0,0 +1,47 @@
+import {getAllColonies} from '../Colony';
+import {log} from '../console/log';
+import {DirectiveSKOutpost} from '../directives/colony/outpostSK';
+import {Directive} from '../directives/Directive';
+import {Overseer} from '../Overseer';
+import {profile} from '../profiler/decorator';
+
+export const ROOMTYPE_SOURCEKEEPER = 'SK';
+export const ROOMTYPE_CORE = 'CORE';
+export const ROOMTYPE_CONTROLLER = 'CTRL';
+export const ROOMTYPE_ALLEY = 'ALLEY';
+
+/**
+ * Empire: Utilities on analyzing the overall empire
+ */
+@profile
+export class EmpireAnalysis {
+
+	static empireMineralDistribution(): {[mineralType: string]: number} {
```

Comment:
pedantic, but methods should be named as verbs. `getEmpireMineralDistribution` or something